locales/en-US/translation.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
If .js is not possible, stripping comments at build is a nice supplement. I'm thinking about the clarity of the keys, comments might be useful.